src/main/java/org/opensearch/agent/tools/SearchAnomalyResultsTool.java

```diff
@@ -130,7 +130,9 @@ public <T> void run(Map<String, String> parameters, ActionListener<T> listener)
             .from(startIndex)
             .sort(sortString, sortOrder);
 
-        SearchRequest searchAnomalyResultsRequest = new SearchRequest().source(searchSourceBuilder);
+        // In the future we may support custom result indices. For now default to the default AD result system indices.
+        String resultIndices = ".opendistro-anomaly-results*";
```

Comment:
Do we have any file that stores constant?

Reply:
Nothing clean exposed from AD plugin. Considered this a one-off so just hardcoded.

Reply:
But good point, I will add to `ToolConstants` file.

Reply:
Updated